lib/plugins/aws/invokeLocal/index.js

```diff
@@ -300,31 +298,43 @@ class AwsInvokeLocal {
     );
   }
 
+  getDockerImageName() {
+    return `sls-docker-${this.getRuntime()}`;
+  }
+
   buildDockerImage(layerPaths) {
     const runtime = this.getRuntime();
-    const imageName = 'sls-docker';
+    const imageName = this.getDockerImageName();
 
-    return new BbPromise((resolve, reject) => {
-      let dockerfile = `FROM lambci/lambda:${runtime}`;
-      for (const layerPath of layerPaths) {
-        dockerfile += `\nADD --chown=sbx_user1051:495 ${layerPath} /opt`;
-      }
-      mkdirp.sync(path.join('.serverless', 'invokeLocal'));
-      const dockerfilePath = path.join('.serverless', 'invokeLocal', 'Dockerfile');
-      fs.writeFileSync(dockerfilePath, dockerfile);
-      this.serverless.cli.log('Building Docker image...');
-      const docker = spawn('docker', [
-        'build',
-        '-t',
-        imageName,
-        `${this.serverless.config.servicePath}`,
-        '-f',
-        dockerfilePath,
-      ]);
-      docker.on('exit', error => {
-        return error ? reject(error) : resolve(imageName);
-      });
-    });
+    mkdirp.sync(path.join('.serverless', 'invokeLocal', runtime));
+    const dockerfilePath = path.join('.serverless', 'invokeLocal', runtime, 'Dockerfile');
+    return fs
+      .accessAsync(dockerfilePath)
```

Comment:
Shouldn't we write docker file in all cases (?) It's cached by runtime, while content of the file is also different per different set of layer paths